src/zenml/integrations/azure/step_operators/azureml_step_operator.py

```diff
@@ -13,50 +13,35 @@
 #  permissions and limitations under the License.
 """Implementation of the ZenML AzureML Step Operator."""
 
-import itertools
-import os
 from typing import TYPE_CHECKING, Dict, List, Optional, Tuple, Type, cast
 
-from azureml.core import (
-    ComputeTarget,
-    Environment,
-    Experiment,
-    ScriptRunConfig,
-    Workspace,
-)
-from azureml.core.authentication import (
-    AbstractAuthentication,
-    ServicePrincipalAuthentication,
-)
-from azureml.core.conda_dependencies import CondaDependencies
+from azure.ai.ml import MLClient, command
+from azure.ai.ml.entities import Environment
+from azure.core.credentials import TokenCredential
+from azure.identity import ClientSecretCredential, DefaultAzureCredential
 
-import zenml
-from zenml.client import Client
-from zenml.constants import (
-    ENV_ZENML_CONFIG_PATH,
-)
-from zenml.environment import Environment as ZenMLEnvironment
+from zenml.config.build_configuration import BuildConfiguration
+from zenml.enums import StackComponentType
+from zenml.integrations.azure.azureml_utils import create_or_get_compute
```

Comment:
Maybe i'm missing something, but both this file and wherever `AzureMLComputeSettings` are defined are missing?

Reply:
I was thinking the exact same thing 😄

Reply:
yes, true, I just added them